### ISSUE TITLE
No documentation for how to delete controllers

### ISSUE THREAD

**Issue by rocifier** [MicrosoftDocs/visualstudio-docs#2896]
How do I delete a controller from the list? I mistyped the name of one and now it's stuck in the list. It also happens to be the first one so it pops up an error when you first open the dialog box. How can I delete controllers from the list?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 6ea6e926-b531-c5a3-e768-f86ff90b6d11
* Version Independent ID: 9fd70783-0997-d758-165a-2c4a33db58ff
* Content: [Configure test agents and test controllers for load tests - Visual Studio](https://docs.microsoft.com/en-us/visualstudio/test/configure-test-agents-and-controllers-for-load-tests?view=vs-2017#feedback)
* Content Source: [docs/test/configure-test-agents-and-controllers-for-load-tests.md](https://github.com/MicrosoftDocs/visualstudio-docs/blob/master/docs/test/configure-test-agents-and-controllers-for-load-tests.md)
* Product: **visual-studio-windows**
* GitHub Login: @gewarren
* Microsoft Alias: **gewarren**

**Comment by gewarren** [comment]
@PBoraMSFT Can you help out with this question?

**Comment by ganesp** [comment]
@rocifier 
This list of recently used controller names is stored in virtual registry managed by Visual Studio.

You can run the following command to remove the registry key which stores the list of recently used controller names.

**Caveats:** 
1) This will clear the entire list.
2) Note that the usage of this tool/command are NOT documented publicly. So the behavior may change in future.

**Command**
"C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\VsRegEdit.exe" delete local HKCU "" EnterpriseTools\QualityTools\RunConfig\RecentlyUsedControllerList

//cc: @PBoraMSFT, @gewarren

**Comment by rocifier** [comment]
Thanks for your reply. This will provide a workaround in the meantime, but it doesn't seem very user friendly. Do you know if there is a task backlogged to add a delete button? The UX for this tool is in a very sad state considering it is a premium feature available to those who invest in visual studio enterprise licenses.

**Comment by PBoraMSFT** [comment]
@rocifier - agree, the experience is not user friendly. 
However, addressing this is not on the backlog though - as we announced deprecation of the load test tools.
https://docs.microsoft.com/en-us/visualstudio/releases/2019/release-notes-preview#test-tools
https://devblogs.microsoft.com/devops/cloud-based-load-testing-service-eol/

@gewarren - can you please close this item? I don't have the permissions.

**Comment by rocifier** [comment]
This is very sad news. The tool has been one of my most loved parts of Visual Studio, one way I have got ahead in my career, and many ways that I have saved my company money. I guess the announcement is right, there are alternatives, and the features look for the most part better than the VS performance tools. This didn't used to be the case, but development on the VS performance tools basically stopped and seems to have been surpassed because of that.
